cluster-service/deploy/openshift-templates/arohcp-service-template.yml

```diff
@@ -368,6 +368,129 @@ objects:
         }
       }
 
+# TODO: parameterize data.platform-identities.yaml after templatize refactor and helm chart conversion
+- apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    name: azure-operators-managed-identities-config
+    namespace: ${NAMESPACE}
+  data:
+    azure-operators-managed-identities-config.yaml: |
+      controlPlaneOperatorsIdentities:
+
+        cloud-controller-manager:
+          minOpenShiftVersion: 4.17
+          maxOpenShiftVersion: 4.17
```

Comment:
Why for all the operators min and max version is set to 4.17? Why not leave max as not set ? The implication of setting max to 4.17 is that when 4.18 is out for example, we'll need to update the version here otherwise we can't really use the config as it is with that version as it'll fail

Reply:
I didn't realize I could leave that field unset.  I am concerned with leaving this unset though, future versions of OpenShift may add required permissions to the CredentialsRequests that were not in prior versions.

Reply:
Yes, the min/max openshift versions fields are optional. 
CS won't complain if you don't set them. 

> I am concerned with leaving this unset though, future versions of OpenShift may add required permissions to the CredentialsRequests that were not in prior versions.

That's a fair point. I think we'll notice it with a failing upgrade / installation (when trying out the new version before they reach out a prod env) to which we'll only need to update the permission. While with the max version being set, we'll need to first update the max version and then only to see that it was a missing permission issue and bump it.

I saw the max version useful for when we know that there is an operator that's only available up to that version. For future version, it has been removed, so for that operator we'll add the max version to guard it. 
The min version follows the same reasoning in my head.

Reply:
I removed the max openshift version fields.